test/commands/scanner/run.test.ts

```diff
@@ -617,11 +617,19 @@ describe('scanner:run', function () {
 			setupCommandTest
 				.command(['scanner:run',
 					'--target', path.join('test', 'code-fixtures', 'invalid-lwc'),
-					'--format', 'csv',
+					'--format', 'json',
 					'--engine', 'eslint-lwc'
 				])
 				.it('LWC Engine detects LWC errors', ctx => {
-					expect(ctx.stdout).to.contain('Invalid public property initialization for "foo"');
+					const results = JSON.parse(ctx.stdout);
```

Comment:
I had a class with some utility methods to do this, but I removed them for now, because I think extending the chai assertions may be a  better path forward. In any case we should invest some time in making our test assertions more consistent with less repetition.